spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
```

Comment:
This testing is fine for now. Testing 3rd party API data can be tricky so these assertions are good.